Sources/Kitura/RouterRequest.swift

```diff
@@ -125,15 +121,10 @@ public class RouterRequest {
     public var remoteAddress: String { return serverRequest.remoteAddress }
 
     /// Parsed Cookies, used to do a lazy parsing of the appropriate headers.
-    private lazy var _cookies: Cookies = { [unowned self] in
-        return Cookies(headers: self.serverRequest.headers)
+    public lazy var cookies: [String: HTTPCookie] = { [unowned self] in
```

Comment:
OK, I don't mind removing it, @shmuelk may have had a reason the code was structured this way though.